explainer.md

```diff
@@ -139,10 +140,61 @@ for (p of purchases) {
 }
 ```
 
+The `listPurchaseHistory` method allows a client to get a list of previous purchases by the user, regardless of current ownership state. Depending on the underlying service provider support, this might be limited to a single purchase record per item.
+
 ## Full API interface
 
+### API v2.1
```

Comment:
Minor nit: It's difficult to see the difference between 2.0 and 2.1 at a glance, which would be useful to double-check that the changes are not breaking. Perhaps it would be useful to have a short summary section here? For example:

#### Summary of changes
 * Add `listPurchases()` method.
 * Add `introductoryPriceCycles` field.

Reply:
Done